addons/uh60_mfd/functions/fnc_storePylons.sqf

```diff
@@ -0,0 +1,21 @@
+/*
+ * vtx_uh60_mfd_fnc_storePylons
+ *
+ * Stores pylon values used in MFDs for recall during reload events
+ *
+ * params (array)[(object) vehicle]
+ */
+
+params ["_vehicle"];
+
+private _pylonArray = [];
+
+{
+	private _index = _x;
```

Comment:
if this is for readability, consider
`_x params ["_index", "", "", "", "_value"];`